frontend/src/routes/(authenticated)/+page.ts

```diff
@@ -12,28 +12,22 @@ export async function load(event: PageLoadEvent): Promise<{ projects: Project[]
   const client = getClient();
   //language=GraphQL
   const results = await client.query(graphql(`
-        query loadProjects($userId: uuid) {
-            projects(where: {ProjectUsers: {userId: {_eq: $userId}}}){
+        query loadProjects {
+            myProjects {
```

Comment:
Did you just change this for curiosity's sake? I'm guessing not and you ran into trouble here? It just seems to me that this should still work (with a slightly different syntax albeit (Maybe just `eq` instead of `_eq` and `users` instead of `ProjectUsers` ).)

But, you know what you're doing 👍

Reply:
basically permissions and the single responsibility principle. I couldn't keep the old way because it would allow users to query any project, even ones they don't belong too. I could have changed the projects query so that server side it conditionally filters based on the user (admin has no filter, all other users can only see their projects), but that would be messy, it would also prevent admins from seeing their list of projects as they would just always see all of them. So instead I made a second query that is explicit in only returning your projects. Now that I think about it I need to lock down the projects query for only admins.